string timestamp
string datestamp
float64 lat
float64 lon
float64 spd_over_grnd
